waspc/cli/Command/Db.hs

```diff
@@ -0,0 +1,50 @@
+module Command.Db
+    ( runDbCommand
+    ) where
+
+import Control.Concurrent.Async (concurrently)
+import Control.Concurrent (Chan, newChan, readChan)
+import Control.Monad.IO.Class (liftIO)
+import Control.Monad.Except (throwError)
+import System.Exit (ExitCode (..))
+
+import StrongPath ((</>))
+import Generator.ServerGenerator.Setup (setupServer)
+import Generator.Job.IO (printJobMessage)
+import qualified Generator.Job as J
+import Command (Command, CommandError(..), runCommand)
+import Command.Compile (compile)
+import Command.Common (findWaspProjectRootDirFromCwd, waspSaysC)
+import qualified Common
+
+runDbCommand :: Command a -> IO ()
+runDbCommand = runCommand . makeDbCommand 
+
+makeDbCommand :: Command a -> Command a
```

Comment:
You could quickly explain what is the point here -> that it ensures some extra stuff is done that is needed for all db commands. Otherwise why would I use it? Maybe even better, you could explain what it actually does -> makes sure code is there and prisma client is generated and so on. Because right now, it is hard to figure that out, since implementation is not doing exactly that, it is setting up whole server, so if I am looking at this first time, it might not be very clear why is this done.